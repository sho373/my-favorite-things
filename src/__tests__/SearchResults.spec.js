import React from 'react';
import { shallow } from 'enzyme';
import {configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import ResultsPage from '../pages/search-results/results.component';

configure({adapter: new Adapter()});

it('should render SearchResults  page component', () => {
    const mockResults = {
        location:{
            state:{
                genreId:"book",
                results:[
                    {
                        Item:{
                            author: "吾峠 呼世晴",
                            largeImageUrl: "https://thumbnail.image.rakuten.co.jp/@0_mall/book/cabinet/6265/9784088816265.jpg?_ex=200x200",
                            publisherName: "集英社",
                            title: "鬼滅の刃 13",
                            itemUrl: "https://books.rakuten.co.jp/rb/15644905/"

                        }
                    }
                ]
            }
        }
    };

    expect(shallow(<ResultsPage props={mockResults}/>)).toMatchSnapshot();
});