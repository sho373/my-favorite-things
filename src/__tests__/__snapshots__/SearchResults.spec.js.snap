// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SearchResults  page component 1`] = `ShallowWrapper {}`;
